### ISSUE TITLE
Invoke Previously Defined Lambdas

### ISSUE THREAD

**Issue by kickroot** [openraven/magpie#171]
Once #170 is complete, we can then take those EC2 instance groupings and (synchronously) invoke lambdas against them. For this first ticket we'll manually ensure these lambdas already exist in the VPCs in question.

Log responses to output.

**Comment by belosh59** [comment]
So I assume I could rely on code for execution of the existing function:
io.openraven.scheduling.states.RunExistingFunction

**Comment by belosh59** [comment]
Invoke already defined lambda and pass the instanceId as the param, and Ip address

**Comment by belosh59** [comment]
Already done.
